packages/manager/modules/pci/src/projects/project/private-networks/private-networks.html

```diff
@@ -0,0 +1,12 @@
+<div class="text-center" data-ng-if="$ctrl.loading">
+    <oui-spinner data-size="m"></oui-spinner>
+</div>
+
+<div data-ng-if="!$ctrl.loading">
+
+    <div data-ui-view="privateNetworkHeaderView"></div>
+
+    <cui-message-container data-messages="$ctrl.messages"></cui-message-container>
+
+    <div data-ui-view="privateNetworkView" class="container-fluid"></div>
```

Comment:
`class="container-fluid"` could be at the component level? Because it will be hard to figure out which classes already apply the parent.